csrc/device_lower/analysis/device_version.cpp

```diff
@@ -14,7 +14,7 @@
 namespace nvfuser {
 
 void MinimumDeviceVersion::dispatch(Val* val) {
-  if (val->dtype() == DataType::BFloat16) {
+  if (val->dtypes() == DataType::BFloat16) {
```

Comment:
```suggestion
  if (val->dtype() == DataType::BFloat16) {
```
**[Suggested by AI]** Verified that the build will pass with this change.